kernel/build-kernel_test.sh

```diff
@@ -58,7 +58,10 @@ build_kernel() {
 
 	echo "Build default kernel"
 	out=$(${build_kernel_sh} build 2>&1)
+	if [ "$(uname -m)" != "ppc64le" ]
```

Comment:
You can use `kata-arch.sh` as `lib.sh` has already set `$kata_arch_sh` to it's path at this point.